app/Modules/User/Views/submission/register.blade.php

```diff
@@ -36,7 +36,7 @@
                         <div class="input-group">
                             <label class="input-group-btn">
                     <span class="btn btn-primary">
-                        Browse&hellip; <input name="file" type="file" style="display: none;">
+                        Browse&hellip; <input name="file" type="file" accept=".doc,.docx,.xml,application/msword,application/vnd.openxmlformats-officedocument.wordprocessingml.document" style="display: none;">
```

Comment:
IMO it's not necessary to add validation on client side, because in laravel it can be handled by controller. I already set for ```doc``` & ```.docx```. to keep the code clean, please remove this changes. Thanks.